SMCKit/SMC.swift

```diff
@@ -617,6 +662,8 @@ public struct SMC {
     - parameter key: The SMC key to check. 4 byte multi-character constant. Must be
                 4 characters in length.
     - returns: valid True if the key is found, false otherwise
+    - returns: IOReturn IOKit return code
```

Comment:
Looks like these got added back in. I removed them because originally, way back, I had thought there should be a `:returns:` line for each return value, since tuple returns were new to Swift. However, single return line is the convention it seems, independent of the number of values.